conf/messages

```diff
@@ -587,31 +587,24 @@ indefiniteAgentRole.HMRC-CBC-NONUK-ORG=an agent
 indefiniteAgentRole.HMRC-PILLAR2-ORG=an agent
 indefiniteAgentRole.noEMA=an agent
 
-consentInformation.p1={0} want to be authorised as your {1} for {2}.
-
-consentInformation.HMRC-MTD-IT.p2=When you have an agent for {0}, you can choose how much support you want from them.
-consentInformation.HMRC-MTD-IT.p3=There are 2 levels of support to choose from:
-consentInformation.HMRC-MTD-IT.li1=a ‘main agent’ can complete most tasks on your behalf
-consentInformation.HMRC-MTD-IT.li2=a ‘supporting agent’ can carry out some tasks, but they cannot submit end-of-year tax returns or see payment information
-consentInformation.HMRC-MTD-IT.guidance=Read the guidance about <a href={0} class="govuk-link" target="_blank" rel="noreferrer noopener">the difference between main agents and supporting agents (opens in new tab)</a>.
-
-consentInformation.HMRC-MTD-IT-SUPP.p2=When you have an agent for {0}, you can choose how much support you want from them.
-consentInformation.HMRC-MTD-IT-SUPP.p3=There are 2 levels of support to choose from:
-consentInformation.HMRC-MTD-IT-SUPP.li1=a ‘main agent’ can complete most tasks on your behalf
-consentInformation.HMRC-MTD-IT-SUPP.li2=a ‘supporting agent’ can carry out some tasks, but they cannot submit end-of-year tax returns or see payment information
-consentInformation.HMRC-MTD-IT-SUPP.guidance=Read the guidance about <a href={0} class="govuk-link" target="_blank" rel="noreferrer noopener">the difference between main agents and supporting agents (opens in new tab)</a>.
+# ________________________________________________________________________________
+# Client consent information
+# ________________________________________________________________________________
+consentInformation.section1.h2=What you need to know first
+consentInformation.section1.p1=An agent can either be ''main agent'' or a ''supporting agent'' when they manage your {0}.
```

Comment:
this now appears to be showing for all regimes when in fact it is only EMA/ITSA related content - this is why we had `HMRC-MTD-IT` or `HMRC-MTD-IT-SUPP` in the previous version of the message keys and in the template tested if they exist before rendering it using `messages.isDefinedAt()` So for those 2 ITSA regimes message keys will exist and the content will show, and for HMRC-MTD-VAT or any other service key, the content will not show as there are no messages defined with those service names in the keys.